src/color-palette.js

```diff
@@ -64,16 +64,23 @@ function generateSuggestedSwatch(fgColor, bgColor, onColorSuggestion) {
     return '';
   }
   let fgColorHex = fgColor.toHexString();
-  console.log(suggestedColors);
-  return <button style={{ color: suggestedColors.fg,  backgroundColor: suggestedColors.fg }}
-                 onClick={() => onColorSuggestion(fgColorHex, suggestedColors.fg)}>@@</button>
+
+  return <button className='selected-color'
```

Comment:
this makes me think we should separate out the selected-color thing (sometime in the future when we feel like it 😄 )